data/json/item_groups.json

```diff
@@ -6857,6 +6857,10 @@
     "items":[
       { "group": "guns_launcher_milspec", "prob": 10 },
       { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "charges-min": 1, "charges-max": 10 },
+      { "item": "bone_human", "prob": 10 "charges-min": 1, "charges-max": 5 },
```

Comment:
Missed comma before charges min here and below.

Reply:
I saw that and fixed it with the seat spawn chance correction